package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.72.0",
+  "version": "7.72.1",
```

Comment:
I ran locally and it made changes to `package-lock.json`. Could you check in those changes, too?